gbpservice/nfp/configurator/test/lb_rpc_sender_v1.py

```diff
@@ -0,0 +1,620 @@
+                     }
+                ]
+        self.members = [
+                {
+                    "admin_state_up": True,
+                    "status": "ACTIVE",
+                    "status_description": None,
+                    "weight": 1,
+                    "address": "42.0.0.11",
+                    "tenant_id": "f6b09b7a590642d8ac6de73df0ab0686",
+                    "protocol_port": 80,
+                    "id": "4910851f-4af7-4592-ad04-08b508c6fa21",
+                    "pool_id": "6350c0fd-07f8-46ff-b797-62acd23760de"},
+                {
+                    "admin_state_up": True,
+                    "status": "ACTIVE",
+                    "status_description": None,
+                    "weight": 1,
+                    "address": "42.0.0.13",
+                    "tenant_id": "f6b09b7a590642d8ac6de73df0ab0686",
+                    "protocol_port": 22,
+                    "id": "76d2a5fc-b39f-4419-9f33-3b21cf16fe47",
+                    "pool_id": "6350c0fd-07f8-46ff-b797-62acd23760de"
+                    }
+                ]
+
+    def get_service_info(self):
+        service_info = {}
+        service_info['pools'] = self.pools
+        service_info['members'] = self.members
+        service_info['vips'] = self.vips
+        service_info['health_monitors'] = self.health_monitors
+        service_info['ports'] = self.ports
+        service_info['subnets'] = self.subnets
+        return service_info
+
+
+class LBRpcSenderV1(object):
+    """ Client side of service manager """
+
+    API_VERSION = '1.0'
+
+    def __init__(self, host=HOST, topic=CONFIGURATOR_TOPIC):
+        self.host = host
+        super(LBRpcSenderV1, self).__init__()
+        target = oslo_messaging.Target(
+                                       topic=topic,
+                                       version=self.API_VERSION)
+        n_rpc.init(cfg.CONF)
+        self.client = n_rpc.get_client(target)
+        self.lb_resources = LBResources()
+        self.context = context.get_admin_context_without_session()
+        self.lb_context = LBContext()
+
+    def _prepare_request_data(self, resource, kwargs):
+        request_data = {
+             'info':   {'version': 'v1',
+                        'service_type': 'loadbalancer'},
+             'config': [{'resource': resource,
+                         'kwargs': kwargs}]
+            }
+        return request_data
+
+    def test_create_pool(self):
+        print 'called test_create_pool'
+        pool_info = {'context': {'service_info':
+                                 self.lb_context.get_service_info()},
+                     'pool': self.lb_resources.pool,
+                     'driver_name': 'loadbalancer'
+                     }
+
+        pool_data = self._prepare_request_data('pool', pool_info)
+        cctxt = self.client.prepare(server=self.host)
+        return cctxt.call(self.context, 'create_network_service_config',
```

Comment:
Are we really making the RPC calls? We need to mock them right? Also, we need not send RPC calls to configurator. Since this is unit testing of LB, we need to send RPCs to LB agent and test.